setup.py

```diff
@@ -8,5 +8,5 @@
     license='',
     author='Jiri Kulda',
     author_email='Kulda12@seznam.cz, jkulda@redhat.com',
-    description='Exhibiting structural change of test metadata.', requires=['ansible']
+    description='Exhibiting structural change of test metadata.', requires=['ansible', 'requests']
```

Comment:
requires should be on its own line like all the other kwargs no?

also isn't it `install_requires` ?